src/schema/fixtures.test.ts

```diff
@@ -11,7 +11,7 @@ const subjectJson = JSON.stringify({ subOverwrite: 'subject', subValue: 'subject
 const rootJson = JSON.stringify({ rootOverwrite: 'root', rootValue: 'root' })
 
 function readBytes(json: string) {
-  return (size: number) => Promise.resolve(new TextEncoder().encode(json))
+  return (size: number) => Promise.resolve(new TextEncoder().encode(json).slice(0, size))
```

Comment:
We either need to overwrite `file.size` or we can just `slice()` to get the full length.

```suggestion
  return (size: number) => Promise.resolve(new TextEncoder().encode(json).slice())
```